astro_data/bright_stars.csv

```diff
@@ -0,0 +1,79 @@
+Nr,NamedStar,LatinDesignation,RA Hr,RA Min,Dec Deg,Dec Min,Magnitude,Const
+0,Acamar,Theta Eridanus,2,58.2,-40,18,3.2,Eri

```

Comment:
the new font can use real greek letters, which is neat to see but this is probably more user friendly